examples/api/open-next.config.ts

```diff
@@ -0,0 +1,25 @@
+import type { OpenNextConfig } from "open-next/types/open-next";
+
+const config: OpenNextConfig = {
+  default: {
+    override: {
+      wrapper: "cloudflare",
```

Comment:
Is this default override necessary ? It seems that you're not using the default server at the moment

Reply:
Good catch, it was used by an other experiment and should be removed from here.